linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -3442,7 +3446,8 @@ where
     });
     let (executed_block0, _) = worker.stage_block_execution(block0).await?;
     let value0 = HashedCertificateValue::new_confirmed(executed_block0);
-    let certificate0 = make_certificate(&committee, &worker, value0.clone());
+    let certificate0 =
+        make_certificate_with_round(&committee, &worker, value0.clone(), Round::MultiLeader(0));
```

Comment:
Comment in line 3455 is outdated. Please fix

Reply:
But it's not, is it? After this block is handled, the ownership is changed to one which has a single super owner, and then the first round _is_ the fast round, with owner 0 as a super owner. Or am I misunderstanding your comment?

Reply:
It's not a fast round anymore. It's `Round::Multileader` (unlike `Round::Fast`)

Reply:
It will be a fast round, though. The previous block executes an ownership change. After that, there will be a super owner, which means the first round will be a fast round - at least, as far as I understand how this works :sweat_smile:

Reply:
Which comment do you mean? Sorry, I can't tell anymore after the code was changed.

`certificate0` is in a multi-leader round. But if this is the certificate that changes the ownership to a super owner, then at future block heights the first round will be `Fast`.